docs/rst/user_guide/conventions.rst

```diff
@@ -16,28 +16,50 @@ Local coordinate definition
 Internally, Eradiate and its radiometric kernel Mitsuba use Cartesian
 coordinates to locate scene objects. Scene construction, when relevant, map the
 X, Y and Z axes to the local East, North and up directions at the origin
-location. Note that this does not mean that Eradiate always uses an East, North,
+location.
+
+.. only:: not latex
+
+   .. image:: fig/cartesian-coordinate-system.svg
+
+Note that this does not mean that Eradiate always uses an East, North,
 up (ENU) coordinate system: this is only relevant when planetary curvature can
 be neglected.
 
 Local illumination and viewing angle definition
 -----------------------------------------------
 
-Internally, Eradiate defines the zenith angle :math:`\theta` as the angular
-distance (in degree or radian) to the local zenith. From this, it follows that:
+Internally, Eradiate also uses the spherical coordinate system with the
+`ISO 80000-2:2019 convention <https://www.iso.org/standard/64973.html>`_
```

Comment:
The spherical coordinate system has little to do with the definition of local illumination and viewing angles. In particular, the radial distance is irrelevant when it comes to the geometry (the distance to the satellite or the Sun does not matter). If you think a paragraph on spherical coordinates is required, I'd like it to be a stand-alone part.

Reply:
I agree, you are completely right, they are separate topics.

Yes, I think it is best if we clarify our conventions for spherical coordinates, given the feedback from @wint3ria -- which lead to eradiate/eradiate-issues#90 -- and our knowledge of the problem of the co-existence of multiple angular conventions. I'll split the content into two sections ; spherical coordinate and local illumination and viewing angles!